### ISSUE TITLE
[Bug] Error socket

### ISSUE THREAD

**Issue by Lennord** [crewpvp/skcrew#10]
sometimes but not all the time, I get this error which displays to me while the paper server is connected to the bungee socket.
```
[02:21:59 WARN]: [Skcrew] Plugin Skcrew v3.2 generated an exception while executing task 19
java.lang.NullPointerException: Cannot invoke "com.lotzy.skcrew.shared.sockets.data.SpigotServer.getPlayers()" because "this.server" is null
	at com.lotzy.skcrew.shared.sockets.data.SpigotPlayer.setServer(SpigotPlayer.java:32) ~[Skcrew.jar:?]
	at com.lotzy.skcrew.shared.sockets.data.SpigotServer.lambda$new$1(SpigotServer.java:30) ~[Skcrew.jar:?]
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]
	at com.lotzy.skcrew.shared.sockets.data.SpigotServer.<init>(SpigotServer.java:30) ~[Skcrew.jar:?]
	at com.lotzy.skcrew.spigot.sockets.SocketClientListener.lambda$onReceivePacket$18(SocketClientListener.java:124) ~[Skcrew.jar:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?]
	at java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1850) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682) ~[?:?]
	at com.lotzy.skcrew.spigot.sockets.SocketClientListener.onReceivePacket(SocketClientListener.java:125) ~[Skcrew.jar:?]
	at com.lotzy.skcrew.spigot.sockets.SocketClient.run(SocketClient.java:81) ~[Skcrew.jar:?]
	at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.20.1.jar:git-Paper-196]
	at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[paper-1.20.1.jar:git-Paper-196]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[paper-1.20.1.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
```

**Comment by Lotzyprod** [comment]
Yea, see this yesterday, I'll try to fix that today

**Comment by Lotzyprod** [comment]
Fixed, update binary jar in last release (3.2), just redownload this
